pkg/kvstore/configurable.go

```diff
@@ -103,3 +104,31 @@ func newKvStoreChainFromConfig(config cfg.Config, logger mon.Logger, name string
 func GetConfigurableKey(name string) string {
 	return fmt.Sprintf("kvstore.%s", name)
 }
+
+var configurableKvStoreLock = sync.Mutex{}
+var ConfigurableKvStores = map[string]KvStore{}
```

Comment:
Why is this map public if you protect it by a mutex? We wouldn't want other people do something with that map without being able to lock it, right? So it would be better if it wouldn't be exported, right?

Reply:
done